### PR TITLE
Added a new command to build relation multipolygon WKTs

### PR DESCRIPTION
**Pull request by adahn6**
### Description:

This PR adds an atlas-shell-tools command to build and validate relation multipolygon WKTs from a set of input Atlases.

### Potential Impact:

None, just a new command!

### Test Results:
------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonCommand.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        return 0;
+    }
+
+    @Override
+    protected int finish()
+    {
+        if (this.relationAtlases.isEmpty())
+        {
+            this.outputDelegate.printlnErrorMessage("no atlases found with relation");
+            return 1;
```
Comment (lucaspcram) [inline src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonCommand.java:111]: Any reason for returning 1 here, and -1 above? As a note: -1 will likely show up as 255 in most shells. Maybe include a note about the different return codes in the description? I know this is a nit :P

File: src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonCommand.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    {
+        if (atlas.relation(this.relationId) != null)
+        {
+            final Optional<Atlas> relationSubAtlas = SubAtlasCreator.softCut(atlas,
+                    entity -> entity.getType().equals(ItemType.RELATION)
+                            && entity.getIdentifier() == this.relationId);
+            if (relationSubAtlas.isPresent())
+            {
+                this.relationAtlases.add(relationSubAtlas.get());
+
```
Comment (lucaspcram) [inline src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonCommand.java:191]: nit: whitespace

File: src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonDescriptionSection.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Given an input directory of Atlas files and a relation ID, find all relation members for the relation
+inside the Atlas files and attempt to create a multipolygon. Assumes the relation should be a 
+multipolygon (e.g. is tagged type->multipolygon). Should the data not construct an OGC or OSM valid
+multipolygon, an error will be logged and the command will halt; should the data be OSM valid but not
+OGC valid, then only an error will be logged. Small relation WKTs will be printed to the console, and 
+all valid relation multipolygons will be saved as WKT and Atlas files to the local directory.
```
Comment (lucaspcram) [inline src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonDescriptionSection.txt:6]: Great documentation.

File: src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonCommand.java (hunk 1 above)
Comment (adahn6) [reply src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonCommand.java:111]: oh whoops, this should be -1

**Comment by adahn6** [comment]
yeah idk what's going on with it there

**Pull request merged**